content/en/docs/reference/config-api/kubeadm-config.v1beta3.md

```diff
@@ -0,0 +1,1459 @@
+---
+title: kubeadm Configuration (v1beta3)
+content_type: tool-reference
+package: kubeadm.k8s.io/v1beta3
+auto_generated: true
+---
+Package v1beta3 defines the v1beta3 version of the kubeadm configuration file format.
+This version improves on the v1beta2 format by fixing some minor issues and adding a few new fields.
+
+A list of changes since v1beta2:
+
+- The deprecated "ClusterConfiguration.useHyperKubeImage" field has been removed.
+  Kubeadm no longer supports the hyperkube image.
+- The "ClusterConfiguration.DNS.Type" field has been removed since CoreDNS is the only supported
+  DNS server type by kubeadm.
```

Comment:
technically this content is out of date:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1beta3/doc.go#L25-L40

which likely means not all the changes we finalized last week in v1beta3 are included here.

sidenote: i'm noticing the "list of changes since" is not rendering @ the godoc for v1beta3:
https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3

Reply:
https://github.com/kubernetes/kubernetes/pull/103495